src/main/java/ch/njol/skript/expressions/ExprTool.java

```diff
@@ -51,46 +51,51 @@
  * @author Peter Güttinger
  */
 @Name("Tool")
-@Description("The item a player is holding.")
-@Examples({"player is holding a pickaxe",
-		"# is the same as",
-		"player's tool is a pickaxe",
-		"player's off hand tool is shield #Only for Minecraft 1.9"})
+@Description("The item an entity is holding in their main or off hand.")
+@Examples({"player's tool is a pickaxe",
+	"player's off hand tool is a shield",
+	"set tool of all player's to diamond sword",
+	"set offhand tool of target entity to a bow"})
 @Since("1.0")
 public class ExprTool extends PropertyExpression<LivingEntity, Slot> {
 	static {
-		Skript.registerExpression(ExprTool.class, Slot.class, ExpressionType.PROPERTY, "[the] (tool|held item|weapon) [of %livingentities%]", "%livingentities%'[s] (tool|held item|weapon)");
+		Skript.registerExpression(ExprTool.class, Slot.class, ExpressionType.PROPERTY,
+			"[the] (0¦(tool|held item|weapon)|1¦(off[ ]hand (tool|item)|shield[ item])) [of %livingentities%]",
+			"%livingentities%'[s] (0¦(tool|held item|weapon)|1¦(off[ ]hand (tool|item)|shield[ item]))");
```

Comment:
```suggestion
			"[the] [(1¦off)(-| )[hand]] (tool|held item|weapon) [of %livingentities%]",
			"%livingentities%'[s] [(1¦off)(-| )[hand]] (tool|held item|weapon)");
```
I don't think I've seen anyone ever using the `shield item` pattern so we can safely remove it

Reply:
I've seen people TRY to use it, but it always fights with the shield item, so you're right, I'll remove it